keylime/cloud_verifier_tornado.py

```diff
@@ -723,7 +724,7 @@ async def invoke_get_quote(agent, need_pubkey):
 
     if response.status_code != 200:
         # this is a connection error, retry get quote
-        if response.status_code == 599:
+        if response.status_code in [500, 599]:
```

Comment:
The issue is that socket timeouts are treated as `500` status codes. If we find another way to distinguish them from other `500` issues I'm happy to change this.

Reply:
I have been seeing a growing number of transient `500` status codes which are exactly sockets timeouts, at scale (with hundreds to thousands of `agents`). It is imperative we retry in case we get the aforementioned status code.